Source/WarpX.cpp

```diff
@@ -1551,7 +1551,7 @@ WarpX::BuildBufferMasks ()
                 const Box& dom = Geom(lev).Domain();
                 const auto& period = Geom(lev).periodicity();
                 tmp.BuildMask(dom, period, covered, notcovered, physbnd, interior);
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Missing `if (amrex::Gpu::notInLaunchRegion())`?

Reply:
I agree with you

Reply:
```suggestion
#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
```